### ISSUE TITLE
SinricPro temperature/humidity sensors issue

### ISSUE THREAD

**Issue by pedrofnm** [kakopappa/sinric#483]
Hello, lately I'm having some refresh problem with my temperature/humidity sensors. They only update every 4 hours or so. Sometimes it takes longer. Or if I reset the module. Dunno what's wrong with it. I used the code on the Arduino ide example (1min refresh time)and I'm using esp01 dht11 module. As you can see in the picture I reseted both sensors but only 1 is refreshing every minute and I bet that In a few minutes it will stop refreshing. Let me know if you need me to post the code here. 
![Screenshot_20210519-221349_Sinric_Pro](https://user-images.githubusercontent.com/64743076/118885568-dc70b300-b8ef-11eb-9f9c-6ee59a7735a6.png)

**Comment by sivar2311** [comment]
SinircPro is located here -> https://github.com/sinricpro

I assume both ESP's running the exact same sketch and have the same circumstances (good power supply, good WiFi signal strength). I also assume that these have already been checked by you. (If you have not checked this yet, please check these conditions first.)

Have you checked the activity-log? https://portal.sinric.pro/activitylog
Do you see any messages in the serial log?

**Comment by pedrofnm** [comment]
Yes, both have good WiFi coverage, same code( except device ID), 1 amp power supply that is more than enough. The activity log only shows that It restore sensor state from the cloud when I reset the board, not updating by it self it seems
![Screenshot_20210520-094844](https://user-images.githubusercontent.com/64743076/118948888-a0ba0580-b950-11eb-90ed-f61a5d63a191.png)

**Comment by sivar2311** [comment]
Please enable serial log and check for any errors.
